assets/prometheus-k8s/prometheus.yaml

```diff
@@ -140,6 +140,8 @@ spec:
   podMonitorNamespaceSelector: {}
   podMonitorSelector: {}
   priorityClassName: system-cluster-critical
+  probeNamespaceSelector: {}
+  probeSelector: {}
```

Comment:
This would allow probe objects to be added to prometheus. Do we want that?

Reply:
Good question will ask Christian if 4.7 is okay for us to enable this!

But I wonder why this was not added before, we had Probes in 4.6 already with bump of Prometheus operator, maybe there is something wrong with our manifests?

/hold
until then

Reply:
If the option is not specified, then it defaults to `nil` so our manifests in 4.6 should be fine.

Reply:
Yep, just wondering how come it wasn't brought in, maybe we missed something else as well? 🤔

Reply:
This is strange indeed. I quickly `jb vendor`'ed dependencies in `release-4.6` and `probeSelector` is definitely declared in `prometheus-crd.libsonnet`. Strange it was not rendered during asset generation.

Reply:
Yes we should look into that! Maybe we just didn't update it correctly in release-4.6, do you now see a diff?

Update: we decided to go ahead with Probes in 4.7 for OSD folks to test.